docs/content/rest-api.md

```diff
@@ -1513,6 +1513,330 @@ The following table summarizes the behavior for partial evaluation results.
 
 > The partially evaluated queries are represented as strings in the table above. The actual API response contains the JSON AST representation.
 
+
+## Health API
+
+The `/health` API endpoint executes a simple built-in policy query to verify
+that the server is operational. Optionally it can account for bundle activation as well
+(useful for "ready" checks at startup).
+
+#### Query Parameters
+`bundles` - Boolean parameter to account for bundle activation status in response. This includes
+            any discovery bundles or bundles defined in the loaded discovery configuration.
+`plugins` - Boolean parameter to account for plugin status in response.
+`exclude-plugin` - String parameter to exclude a plugin from status checks. Can be added multiple
+            times. Does nothing if `plugins` is not true. This parameter is useful for special use cases
+            where a plugin depends on the server being fully initialized before it can fully intialize
+            itself.
+
+#### Status Codes
+- **200** - OPA service is healthy. If the `bundles` option is specified then all configured bundles have
+            been activated. If the `plugins` option is specified then all plugins are in an OK state.
+- **500** - OPA service is not healthy. If the `bundles` option is specified this can mean any of the configured
+            bundles have not yet been activated. If the `plugins` option is specified then at least one
+            plugin is in a non-OK state.
+
+> *Note*: The bundle activation check is only for initial bundle activation. Subsequent downloads
+  will not affect the health check. The [Status](../management-status)
+  API should be used for more fine-grained bundle status monitoring.
```

Comment:
This could use an info box, too:

```suggestion
{{< info >}}
The bundle activation check is only for initial bundle activation. Subsequent downloads
will not affect the health check. The [Status](../management-status)
API should be used for more fine-grained bundle status monitoring.
{{< /info >}}
```